vc_assistance/indico_vc_assistance/plugin.py

```diff
@@ -0,0 +1,80 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import session
+from flask_pluginengine import render_plugin_template, url_for_plugin
+
+from indico.core import signals
+from indico.core.plugins import IndicoPlugin
+from indico.modules.events.requests.models.requests import Request, RequestState
+from indico.web.forms.base import IndicoForm
+from indico.web.forms.fields import PrincipalListField
+from indico.web.menu import TopMenuItem
+
+from indico_vc_assistance import _
+from indico_vc_assistance.blueprint import blueprint
+from indico_vc_assistance.definition import VCAssistanceRequest
+from indico_vc_assistance.util import has_room_with_vc_attached, is_vc_support
+from indico_vc_assistance.views import WPVCAssistance
+
+
+class PluginSettingsForm(IndicoForm):
+    managers = PrincipalListField(_('Managers'), groups=True,
+                                  description=_('List of users who can request videoconference assistance.'))
+    vc_support = PrincipalListField(_('Videoconference support'), groups=True,
+                                    description=_('List of users who can view the list of events with videoconference '
+                                                  'assistance.'))
+
+
+class VCAssistanceRequestPlugin(IndicoPlugin):
+    """Videoconference Assistance Request
+
+    Provides a service request where participants can ask for their
+    event to have videoconference assistance.
+    """
+
+    configurable = True
+    settings_form = PluginSettingsForm
+    acl_settings = {'managers', 'vc_support'}
+
+    def init(self):
```

Comment:
Do we also need to grant VC support access to the event (if yes, check `_can_access_event` of the audiovisual request plugin)? Or is the information we provide in the service list enough?

@pferreir

Reply:
I'd say we provide read access.